### ISSUE TITLE
ads coming through as of 8/8/24

### ISSUE THREAD

**Issue by gendriga** [mgziminsky/FacebookTrackingRemoval#88]
Please let me know what info or diags I can supply. Thanks!

![Screenshot 2024-08-08 092350](https://github.com/user-attachments/assets/4b4c8ca0-2585-4d07-bc8e-49cabb7fcb35)

**Comment by mgziminsky** [comment]
I just started seeing these as well. Unfortunately, they don't seem to have anything to distinguish them from regular posts in their content or metadata that I can find. I'll keep looking into it, but unless fb changes something, there might be no solution this time

**Comment by mgziminsky** [comment]
Possible solution via uBlock Origin: [r/uBlockOrigin](https://farside.link/libreddit/r/uBlockOrigin/comments/1en2dfv/facebook_started_to_shows_pages_i_never_followed)

Since that is doing the blocking by modifying the actual network requests, it probably isn't something I'll add to this extension. If uBlock doesn't roll their filter out, I'll consider adding it here, but it would only work in Firefox due to chrome crippling extensions with manifest v3.

**Comment by mgziminsky** [comment]
@gendriga Is this still an issue for you? I haven't seen these on my end for a while. If FB has stopped serving them, or uBlockOrigin has started blocking them in a standard filter list, this issue can be closed.

**Comment by miloslavnosek** [comment]
The add-on has been working amazingly well up until today, when it seems to have stopped working altogether :( Now, 80% of my feed is filled with random ads. From the Reddit link, only the last filter mentioned in the comments seems to do anything, but it’s not really a filter—it basically just redirects you to the feed page from the main Facebook page.  

Hope we'll get a proper fix. At any rate, this addon has been godsent.

**Comment by mgziminsky** [comment]
Yep.. Literally the night I posted asking about it, it started happening again. The same issue with identifying them as before applies though unfortunately. I tried to see if there was anything special to identify them this time, but by the time the extension sees it, it is indistinguishable from a normal post. Hopefully this is just a temporary test on the FB end, or one of the ublock filter lists adds a fix soon.

**Comment by Catullus16** [comment]
I'm assuming Facebook is updating the way ads work again since they're starting to come through again as of the beginning of the month. This is with the latest version of Firefox. Let me know if you need any screenshots or additional data.
